app/assets/javascripts/modules/namespaces/components/new-form.js

```diff
@@ -65,40 +68,28 @@ export default {
       name: {
         required,
         format(value) {
```

Comment:
I think we can rename this to a more generic thing instead of `format`. We are validating everything at once.